script/entrypoint

```diff
@@ -9,6 +9,20 @@ if [ -n "$GCLOUD_SERVICE_KEY" ]; then
     # See https://cloud.google.com/docs/authentication/production
     export GOOGLE_APPLICATION_CREDENTIALS="${GOOGLE_APPLICATION_CREDENTIALS:-/tmp/gcp.json}"
     echo "$GCLOUD_SERVICE_KEY" > "$GOOGLE_APPLICATION_CREDENTIALS"
+    gcloud --quiet auth activate-service-account --key-file "$GOOGLE_APPLICATION_CREDENTIALS"
+    python -W ignore -c 'import google.auth; print("project = ", google.auth.default()[1])' >> ~/.config/gcloud/configurations/config_default
+    cat > ~/.bigqueryrc << EOF
+--project_id=$(gcloud config get-value project)
+--dataset_id=telemetry
+[mk]
+--use_legacy_sql=false
+[query]
+--use_legacy_sql=false
+--replace=true
+--max_rows=0
+[update]
+--use_legacy_sql=false
+EOF
```

Comment:
turns out the cli tools completely ignore `GOOGLE_APPLICATION_CREDENTIALS`, so this adds required credential configs and sets some better defaults for `.bigqueryrc`.